### PR TITLE
Update set-scheduling-policies.md

### PR DESCRIPTION
**Pull request by msbemba**
updated the document to reflect the minimum lead time reflects regular hours

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9964

File: microsoft-365/bookings/set-scheduling-policies.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ This table explains the various scheduling policies available on the booking pag
...
-| Lead time in hours | You build your staffing plan based on the appointments that are scheduled so it's important to know in advance how many customers are coming in for service on any particular day. The lead time policy enables you to specify the number of hours in advance that customers must book or cancel an appointment. |
+| Lead time in hours | You build your staffing plan based on the appointments that are scheduled so it's important to know in advance how many customers are coming in for service on any particular day. The lead time policy enables you to specify the number of hours in advance that customers must book or cancel an appointment. Minimum lead time considers regular hours and not business hours.|
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/set-scheduling-policies.md]:
```suggestion
| Lead time in hours | You build your staffing plan based on the appointments that are scheduled so it's important to know in advance how many customers are coming in for service on any particular day. The lead time policy enables you to specify the number of hours in advance that customers must book or cancel an appointment. The minimum lead time is based on regular hours and not business hours.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Comment by yogkumgit** [comment]
@msbemba gentle reminder

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@kwekuako Pease merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Gentle reminder

**Pull request merged**